tests/mongodb-test.js

```diff
@@ -279,6 +281,23 @@ describe('MongoDB', () => {
 
 			assert.deepStrictEqual(fields._id, ObjID);
 		});
+
+		it('should not reject when MongoClient can\'t reject cause is in cache the const clients is used', async () => {
```

Comment:
Este test esta mal por 2 motivos.
1. Esta en un describe incorrecto `prepareFields()`, debería estar en `checkConnection()`
2. Esta mal hecho, solo funciona cuando se corren los tests anteriores, no es independiente. Esto se puede probar con `it.only('should...')` 
2.1. Para solucionarlo sugiero hacer 2 `checkConnections()` y validar que se llame 1 vez sola por ej

Reply:
Reformular la descripción del it(), no se entiende, tiene que ser mas fácil.
Por ej. 'should cache connection and call connect method just once when multiple checkConnection called'